packages/widget-v2/src/pages/SwapExecutionPage/SwapExecutionPage.tsx

```diff
@@ -182,6 +182,8 @@ export const SwapExecutionPage = () => {
     }
   }, [connectRequiredChains, lastOperation.signRequired, mutate, route?.destAssetChainID, route?.estimatedRouteDurationSeconds, setCurrentPage, setManualAddressModal, swapExecutionState, theme.success.text]);
 
+  const SwapExecutionPageRoute = simpleRoute ? SwapExecutionPageRouteSimple : SwapExecutionPageRouteDetailed;
```

Comment:
since you do this please combine `SwapExecutionPageRouteDetailedProps` and `SwapExecutionPageRouteSimpleProps` into one interface.